pymeasure/display/browser.py

```diff
@@ -26,14 +26,15 @@
 
 from os.path import basename
 
-from .Qt import QtCore, QtGui
+from .Qt import QtCore, QtWidgets
+from pyqtgraph.Qt import QtGui
```

Comment:
Is that last line needed?

Reply:
Also, I'd rather you import that from the local `.Qt` module, then we keep one/only a few entrypoints where we get stuff from pyqtgraph. This will make any future tweaks simpler.

Reply:
Here actually my inexperience with flake8 and github shows... :-(
Do warnings lead to a failed check? If we leave the import of QtGui in `Qt.py` flake8 complains that it is not used in this file. That's why I moved to the local import. Local imports are required in certain places as not all the Widgets of QtGui are now in QtCore, e.g. QPixmap is still in QtGui, this is why in some modules, the `QtGui` import is still required.

Reply:
> Do warnings lead to a failed check?

Yes, this is our way to keep the codebase clean(ish).

> flake8 complains that it is not used in this file.

You can silence this in a line by adding a comment `# noqa: Exxx` (with the associated error code), as here it is OK because we use this to expose the API we need.

Reply:
OK, thanks for the help! Should be implemented now with silencing line and import from `.Qt` module